cmd/infra/powervs/destroy.go

```diff
@@ -492,8 +492,20 @@ func destroyPowerVsCloudConnection(ctx context.Context, options *DestroyInfraOpt
 			}
 		}
 	} else {
+
+		deleteCC := func(id string) error {
```

Comment:
nit: deleteCloudConnection it's easier to read for me than deleteCC

Reply:
done.